### PR TITLE
feat: add DoubleIcon in ExpenseLine header to open selection mode

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#72131

### CODE REVIEW

File: packages/apps/hr/src/i18n/fr.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "Hr_Expenses": "Notes de frais",
   "Hr_NewExpenseLine": "Nouvelle ligne de frais",
   "Hr_NewExpense": "Créer une nouvelle note de frais",
+  "Hr_OpenSelectionMode": "Ouvre le mode de selection",
```
Comment (lme-axelor) [inline packages/apps/hr/src/i18n/fr.json:9]: "Ouvrir... ...sélection."

File: packages/apps/hr/src/screens/ExpenseLinesListScreen.js [hunk 2]
```diff
@@ -72,14 +77,41 @@ const ExpenseLinesListScreen = ({navigation}) => {
...
           onPress: () => navigation.navigate('ExpenseLineFormScreen', {}),
           showInHeader: true,
         },
+        {
+          key: 'openSelectionMode',
+          order: 10,
+          iconName: 'user',
+          hideIf: isSelectionMode,
+          title: I18n.t('Hr_OpenSelectionMode'),
+          iconColor: Colors.primaryColor.background,
```
Comment (lme-axelor) [inline packages/apps/hr/src/screens/ExpenseLinesListScreen.js:94]: iconColor is not really useful here

File: packages/apps/hr/src/screens/ExpenseLinesListScreen.js [hunk 3]
```diff
@@ -72,14 +77,41 @@ const ExpenseLinesListScreen = ({navigation}) => {
...
+          onPress: () => {
+            handleModeChange(null);
```
Comment (lme-axelor) [inline packages/apps/hr/src/screens/ExpenseLinesListScreen.js:96]: dirty 👀

```
 const handleModeChange = (itemId = null) => {
    setIsSelectionMode(current => {
      const mode = !current;

      if (mode && itemId != null) {
        setSelectedItems([itemId]);
      } else {
        setSelectedItems([]);
      }
      return mode;
    });
  };
```

and then : 

```
onPress: handleModeChange,
```

**Review by vhu-axelor — APPROVED** [review]
Work well, nice! 🤩

**Pull request merged**